README_new.md

```diff
@@ -0,0 +1,132 @@
+
+As such, these are the main areas the iOS team will focus our assessment of candidates on:
+
+- Technical expertise
+ - Languages (Swift, Objective-C) and platforms (iOS, watchOS, tvOS, macOS)
+ - Architectural design patterns (MVC, MVVM, Viper, etc)
+ - Knowledge of iOS APIs
+ - Knowledge of the development cycle
+ - Knowledge of reactive programming
+ - Interview demo project
+- Interest in sharing one's knowledge
+- Interest in learning new knowledge
+- Interest in keeping oneself up to date
+- Ability to think and work independently
+- Ability to work in a large team
+- Professional and personal maturity
+- Overall attitude
+- Likelihood of fitting in well with the company's culture
+- Is the candidate someone we would like to work with on a daily basis?
+
+### The iOS recruitment process
+
+Our recruitment process is different between candidates who are interested in working remotely and candidates who are interested in working on-site at our office in London, in **two** ways:
+- remote candidates are *required* to be proficient in **reactive programming** (`Reactive Swift`/`Reactive Cocoa` or `RxSwift`) at an above-average level
+- remote candidates go through a longer technical interview process:
+    * on-site: initial call -> demo -> 3-hour face-to-face technical interview
+    * remote: initial call -> demo -> one or more Skype interviews -> 1 working day with us (at our expense)
+
+As indicated above, we expect all candidates to submit an "interview demo project" but, in order to be flexible, we provide 3 different ways in which a candidate can submit a demo of their work. **Only one of these is required**.
+
+1. [Do the babylon demo project](TODO)
+2. [Send us a project you've already done](TODO)
+3. [Show us some open source work you did](TODO)
+
+A demo project should be an example of one's **best** work. We don't just want to see a technically good project; we want to be *wowed* by a candidate's techincal skills and creativity.
+
+If we think the demo project is interesting enough, we'll proceed to a face-to-face step, either on Skype (for remote candidates) or in person (for on-site candidates).
+
+It typically starts with a discussion around the candidate's project. We normally ask the candidate to implement a new feature (even if it's just using pseudo-code), or to refactor and unit-test a particular component, and in general we aim to have an informal discussion about the candidate's approach.
+
+We may also, or instead of, focus on more technical aspects and ask a number of highly technical questions. As indicated earlier, we have open-sourced these questions as well since we believe that a fully open recruitment process gives all candidates a fairer chance to prepare, and makes our assessment uniform across all candidates.
+
+Above all, be yourself! 🌈
+
+Thank you for taking the time to apply to babylon health. 😄
```

Comment:
These two lines' mood are in stark contrast with the previous paragraphs. I'd suggest collapsing them in

`+Above all, be yourself! And thank you for taking the time to apply to babylon health 🌈

Reply:
The entire text up to this point has been impersonal and about "a candidate" while the text here is very personal and directed to the person reading the page. I'm not sure how to marry together these two directions. Ignoring this dichotomy, this is the best I've got for putting the sentences together:

> Above all, be yourself and thank you for taking the time to apply to babylon health! 😄